src/lib/components/AgGrid.react.js

```diff
@@ -161,6 +161,11 @@ DashAgGrid.propTypes = {
                 PropTypes.bool, PropTypes.arrayOf(PropTypes.object)
         ]),
 
+    /**
+     * If true, the internal method updateRows() will be called
+     */
+    enableUpdateRows: PropTypes.arrayOf(PropTypes.object),
```

Comment:
I'm not sure I understand, can you give an example of how to use this? Eventually we want all these examples to be in the form of dash testing test cases, ie you set up an app and then interact with it and the test says what should change ("click on this button and the third row changes to have this new data in it but the other rows are unchanged") but until we get that pattern going maybe you can just add an app file into the `tests/` dir? (just don't start its name with `test`, so we know it'll need converting to a test case later)

Reply:
@alexcjohnson, do you want me to keep the rest of the test as is, or just remove the other stuff and leave only the table?

Reply:
you mean the other files in that dir? I'd leave those there for now, just add a new file that uses this feature and has a comment describing the actions you can do in the app and what their result is supposed to be.

Reply:
Added a test version for adding, updating, and removing rows.

Adding rows gets glitchy due to possibly duplicated rows, which goes berserk.

Reply:
@alexcjohnson, I've made quite a few adjustments, as well as testing using clientside_callbacks to see if it bucks.

Reply:
OK I see now. This prop isn't _enabling_ update, it _is_ the update.

Can we then rename it to just `updateRows`, with a description something like:

> Provide new data to one or more existing rows. The rows to be replaced are the ones with matching row IDs, based on the field specified by `setRowId`.

And with that in mind actually, can we change the prop `setRowId` to `rowIdField` (including in the description above), with a description something like:

> The field within each row of `rowData` that will be used as the ID when identifying rows for updates.

Reply:
Yes. We could do that. I was just using setRowId because it correlates to getRowId in ag-grid